internal/util/problem_json.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import (
+	"github.com/gin-gonic/gin"
+
+	"github.com/free5gc/openapi/models"
+)
+
+func GinProblemJson(c *gin.Context, problemDetails *models.ProblemDetails) {
+	c.JSON(int(problemDetails.Status), problemDetails)
+	c.Writer.Header().Set("Content-Type", "application/problem+json")
```

Comment:
Should `c.Writer.Header().Set()` performs before the `c.JSON()`?
Please help to check, tks!

Reply:
Yes. `Header().set()` will replace the existing value, which is `Content-Type` in this case. We want to replace `Content-Type: application/json` with `Content-Type: application/problem+json` to align with api spec of r17.

See more details [here](https://pkg.go.dev/net/http#Header.Set)